src/react/components/MnemonicGenerationFlow/3-VerifyMnemonic.tsx

```diff
@@ -24,14 +24,14 @@ const VerifyMnemonic: FC<VerifyMnemonicProps> = (props): ReactElement => {
   return (
     <Grid container direction="column" spacing={3}>
       <Grid item xs={12}>
-        Please retype your Secret Recovery Phrase here to make sure you have it saved correctly.
+        Please retype your Secret Recovery Phrase here to make sure you have it saved correctly. Mistakes could cause a loss of funds.
```

Comment:
Saying "mistakes could cause a loss of funds" at this point is a little confusing, because it's a validation step and there are no funds yet, and mistakes would prevent the user from continuing.  I think the "mistakes..." text would be better in the `2-GenerateMnemonic` step just prior to this.

Reply:
Makes sense - done!